actionpack/lib/action_dispatch/http/uri.rb

```diff
@@ -0,0 +1,246 @@
+# frozen_string_literal: true
+
+require "active_support/core_ext/module/attribute_accessors"
+require "uri/generic"
+
+class ActionDispatch::Http::URI
```

Comment:
I think we almost universally prefer expanded module definitions. We should probably have a cop defined for that.